prime-router/build.gradle.kts

```diff
@@ -100,7 +101,7 @@ fun addVaultValuesToEnv(env: MutableMap<String, Any>) {
 
 defaultTasks("package")
 
-val ktorVersion = "1.6.8"
+val ktorVersion = "2.0.3"
```

Comment:
the major upgrade to [ktor](https://github.com/CDCgov/prime-reportstream/pull/5329) is delayed due to possible breaking changes

Reply:
Ktor is only used in SOAP transport and REST transport. Both pass integration tests, and will be validated in staging. Are there other risks we need to mitigate?

Reply:
@snesm Tom has addressed these changes. Please approve the PR.